utility/doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contribution guide
+
+- For reference, [here is the detailed version](https://github.com/pokt-network/pocket/blob/main/utility/doc/E2E_FEATURE_PATH_TEMPLATE.md#e2e-feature-implementation)
```

Comment:
```suggestion
- For reference, [here is the detailed version](https://github.com/pokt-network/pocket/blob/main/utility/doc/E2E_FEATURE_PATH_TEMPLATE.md#e2e-feature-implementation).
```

Reply:
Thank you for the review. This line is getting deleted as the doc is being moved to a section of the e2e feature template doc.